conans/client/cmake.py

```diff
@@ -10,6 +10,23 @@
 from conans import tools
 import os
 import platform
+from conans.util.log import logger
+
+# Deprecated in 0.22
+deprecated_conanfile_param_message = '''
```

Comment:
I don't know why this message is shown in the diff... it was already in develop! right?